### PR TITLE
Static Pages

### PR DESCRIPTION
**Pull request by Iskandar19**

### CODE REVIEW

**Comment by Iskandar19** [comment]
ready

File: app/assets/stylesheets/static_pages.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (huongnt-2545) [inline app/assets/stylesheets/static_pages.scss:1]: delete this file

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (huongnt-2545) [inline app/helpers/static_pages_helper.rb:1]: delete this file if it's not useful

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
-  <body>
+<body>
+  <%= render "layouts/header" %>
+  <div class="container">
     <%= yield %>
-  </body>
-</html>
+  </div>
+
+</body>
```
Comment (huongnt-2545) [inline app/views/layouts/application.html.erb:18]: Missing the close tag `</html>` at the end

**Review by huongnt-2545 — COMMENTED** [review]
ignore folder `node_modules/` in `.gitignore` file

**Comment by Iskandar19** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
-    <%= yield %>
+  <%= render "layouts/header" %>
+   <div class="container">
+     <%= yield %>
+   </div>
```
Comment (anlvq-1058) [inline app/views/layouts/application.html.erb:16]: indent please

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by Iskandar19** [comment]
ready

File: Gemfile [hunk 5]
```diff
@@ -21,12 +21,13 @@ gem 'jbuilder', '~> 2.7'
...
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'bootstrap-sass', '3.4.1'
 # Use Active Storage variant
 # gem 'image_processing', '~> 1.2'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
+gem 'thor', '~> 1.1.0'
```
Comment (anlvq-1058) [inline Gemfile:30]: may be you don't need this gem, try to remove it

File: test/controllers/static_pages_controller_test.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/controllers/static_pages_controller_test.rb:1]: delete this file please

**Review by anlvq-1058 — COMMENTED** [review]
cmt

File: Gemfile (hunk 5 above)
Comment (Iskandar19) [reply Gemfile]: removed sir, everything still worked fine as of now.

**Comment by Iskandar19** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 7]
```diff
@@ -0,0 +1,40 @@
+/* universal */
+
+body {
+padding-top: 60px;
```
Comment (huongnt-2545) [inline app/assets/stylesheets/custom.scss:4]: wrong indent
Fix the same error in this file
```suggestion
  padding-top: 60px;
```

File: app/views/layouts/_header.html.erb [hunk 8]
```diff
@@ -0,0 +1,12 @@
+ <header class="navbar navbar-fixed-top navbar-inverse">
+    <div class="container">
```
Comment (huongnt-2545) [inline app/views/layouts/_header.html.erb:2]: wrong indent
```suggestion
   <div class="container">
```

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -1,16 +1,21 @@
...
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
+    <%= render "layouts/header" %>
+    <div class="container">
+      <% flash.each do |message_type, message| %>
+        <div class="alert alert-<%= message_type %>"><%= message %></div>
+      <% end %>
     <%= yield %>
```
Comment (huongnt-2545) [inline app/views/layouts/application.html.erb:18]: wrong indent
```suggestion
      <%= yield %>
```

**Review by huongnt-2545 — COMMENTED** [review]
please remove cache folder node_modules by command
```
git rm -rf --cached node_modules/
```

**Review by huongnt-2545 — COMMENTED** [review]
please remove cache folder node_modules by command
```
git rm -rf --cached node_modules/
```

**Comment by Iskandar19** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**